Formula/h/hexd.rb

```diff
@@ -0,0 +1,19 @@
+class Hexd < Formula
+  desc "Colourful, human-friendly hexdump tool"
+  homepage "https://github.com/FireyFly/hexd"
+  url "https://github.com/FireyFly/hexd/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "de0db7205c8eb0f170263aca27f5d48963855345bc79ba4842edd21a938d0326"
+  license "MIT"
+
+  def install
+    inreplace "Makefile", "install -D", "install"
```

Comment:
Why?

Reply:
@SMillerDev Makefile assumes GNU `install`, which is not the case on macOS.